client/state/action-types.js

```diff
@@ -358,6 +358,8 @@ export const HELP_TICKET_CONFIGURATION_REQUEST_FAILURE =
 export const HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS =
 	'HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS';
 export const HISTORY_REPLACE = 'HISTORY_REPLACE';
+export const HOME_CARDS_REQUEST = 'HOME_CARDS_REQUEST';
```

Comment:
We ended up changing the endpoint name to /home/layout, so maybe `HOME_LAYOUT_REQUEST` and `HOME_LAYOUT_SET`?

Reply:
5780180ae1dbc11bd1dd7677c01b7230235902f5